core-plugins/src/main/java/co/cask/hydrator/plugin/common/Properties.java

```diff
@@ -127,6 +127,7 @@
    */
   public static class Table {
     public static final String NAME = "name";
+    public static final String EXPLORE_NAME = "exploreName";
```

Comment:
the other properties in this class use the actual dataset properties, so we should be consistent and use ExploreProperties.PROPERTY_EXPLORE_TABLE_NAME here